keps/prod-readiness/sig-node/4800.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 4800
 alpha:
   approver: "@soltysh"
+beta:
+  approver: "@mrunalp"

```

Comment:
this needs to be one of PRR team member. If unsure just use the reviewer which handled the alpha stage